neurons/old/version.py

```diff
@@ -0,0 +1,3 @@
+__version__ = "0.0.1"
+version_split = __version__.split(".")
```

Comment:
Please, remove this file

Reply:
Should I remove the whole folder then ? It's empty.